app/components/windows/Main.vue

```diff
@@ -1,53 +1,59 @@
 <template>
-<div class="main" :class="theme" id="mainWrapper" @drop="onDropHandler">
-  <title-bar :title="title" />
-  <div class="main-spacer" :class="{ 'main-spacer--error': errorAlert }"></div>
-  <news-banner/>
-  <div
-    class="main-contents"
-    :class="{
-      'main-contents--right': renderDock && leftDock && hasLiveDock,
-      'main-contents--left': renderDock && !leftDock && hasLiveDock }">
-    <div class="live-dock-wrapper" v-if="renderDock && leftDock">
-      <live-dock :onLeft="true" />
-      <resize-bar
-        v-if="!isDockCollapsed"
-        class="live-dock-resize-bar live-dock-resize-bar--left"
-        position="right"
-        @onresizestart="onResizeStartHandler"
-        @onresizestop="onResizeStopHandler"
-      />
-    </div>
-
-    <div class="main-middle" :class="mainResponsiveClasses" ref="mainMiddle">
-      <resize-observer @notify="handleResize"></resize-observer>
-
-      <top-nav v-if="(page !== 'Onboarding') && !showLoadingSpinner" :locked="applicationLoading"></top-nav>
-      <apps-nav v-if="platformApps.length > 0 && (page !== 'Onboarding')"></apps-nav>
-
-      <component
-        class="main-page-container"
-        v-if="!showLoadingSpinner"
-        :is="page"
-        :params="params"/>
-      <studio-footer v-if="!applicationLoading && (page !== 'Onboarding')" />
-    </div>
-
-    <div class="live-dock-wrapper" v-if="renderDock && !leftDock">
-      <resize-bar
-        v-if="!isDockCollapsed"
-        class="live-dock-resize-bar"
-        position="left"
-        @onresizestart="onResizeStartHandler"
-        @onresizestop="onResizeStopHandler"
-      />
-      <live-dock class="live-dock" />
+  <div class="main" :class="theme" id="mainWrapper" @drop="onDropHandler">
```

Comment:
Why was this indentation change made?  This is a fragile component and it's difficult to see if anything changed.  Can you please back this out?

Reply:
Done.